pom.xml

```diff
@@ -41,7 +41,7 @@ under the License.
         <url>https://issues.apache.org/jira/browse/NETBEANSINFRA</url>
     </issueManagement>
     <prerequisites>
-        <maven>3.0.5</maven>
+        <maven>3.6.3</maven>
```

Comment:
Is 3.6.3 the first maven release, that defaults to https as download url from maven central?

Reply:
I do not work, I think it's the latest, it is the one I tested and it worked, so I think it is a good value.

In NetBeans you can ignore that version number, if I remember correctly.

Reply:
3.2.3 seems to be the first version that uses HTTPS

Reply:
Ok - I reread what prerequisites does and was reminded, that if you really want to enforce something maven-enforcer is the the right thing to use: "It is checked with Maven 2, it is not any more in Maven 3.". Lets keep it as is.